configuration/transitions.md

```diff
@@ -191,6 +191,30 @@ If you have changed from the default contact hierarchy you will need to specify
 }
 ```
 
+###### Specific Parent *as of 3.8.0*
+
+By default, the newly created person will have the same parent as the report submitter. 
+A different parent may be selected by providing a location for the parent id. This field should
+contain the `place_id` of the place in question. 
+If the selected parent is invalid - does not exist or does not respect the configured hierarchy
+ - the report is rejected as invalid and the person document is not created. As such
+ , `report_accepted` event should check if the report has a `patient` property (or similar).
+
+```json
+ {
+     "params": "{ \"parent_id\": \"parent_id\" }"
+ }
+```
+
+###### Events
+
+* `parent_field_not_provided` - triggered when the report does not have the required parent_id field
+* `parent_invalid` - triggered when selected parent is invalid
```

Comment:
May be good to add what leads to an "invalid" parent. Is it that they are not at the right place in the hierarchy, or are a person?

Reply:
Above it mentions that invalid is when the patient `does not exist or does not respect the configured hierarchy`. We could include something similar here, and perhaps what not respecting the configured hierarchy means.

Reply:
I added explanations for what `parent_invalid` means.